storage/immunitycache/cache_test.go

```diff
@@ -302,6 +302,13 @@ func TestImmunityCache_ClearConcurrentWithRangeOverChunks(t *testing.T) {
 	wg.Wait()
 }
 
+func TestImmunityCache_CloseDoesNotPanic(t *testing.T) {
```

Comment:
this does not test the code does not panic

Reply:
Renamed.